nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -133,8 +133,9 @@ in
           will set up automatically for each build.  This prevents
           impurities in builds by disallowing access to dependencies
           outside of the Nix store. This isn't enabled by default for
```

Comment:
The sentence looks a bit out of context after mentioning performance.
How about:

This prevents impurities in builds by disallowing access to dependencies outside of the Nix store with network- and mount namespaces.